ynput/OpenPype
# Maya: Supports for additional Job Info and Plugin Info in deadline submission

## Changelog Description
This PR is to resolve some of the attributes such as MTOA's `ArnoldVerbose` are not preserved on farm and users can use the project settings to add the attributes back to either job or plugin Info.

## Additional info
n/a

## Testing notes:
1. Go to Openpype Settings -> `project_settings/deadline/publish/MayaSubmitDeadline/pluginInfo`
2. Add `{"ArnoldVerbose": 2}`
![image](https://github.com/ynput/OpenPype/assets/64118225/0c7424ef-c790-4b73-9510-d799c2e7c253)

3. Launch Maya via launcher with your rendering scene
4. Make sure your current renderer is Arnold
5. Publish
